lib/ruby_lsp/requests/document_symbol.rb

```diff
@@ -220,6 +248,33 @@ def create_document_symbol(name:, kind:, range_node:, selection_range_node:)
 
         symbol
       end
+
+      sig { params(node: SyntaxTree::Command).void }
+      def parse_test(node)
+        return unless node.block
+
+        argument = node.arguments.parts.first
+        return unless argument.is_a?(SyntaxTree::StringLiteral) && node.arguments.parts.size == 1
+
+        name_node = argument.parts.first
+        return unless name_node
+
+        full_name = argument.parts.map do |part|
```

Comment:
What are the parts in this case? Would we have a single string argument?

Reply:
Then do we need to map? Can't we just access the first item?

Reply:
I added the `map` so it could handle strings with interpolation, e.g. we could show `"valid for #{category} something"` as `"valid for category { } something"`. Without the map, it would only show `"valid for category"`.

 I'm on the fence about whether it's worthwhile though.

Reply:
Oh, I understand now what the curly braces are. In that case, I agree with the `map`, but instead of using curly braces it might look better if we use a fake object notation. Something like
```ruby
"valid for category #<Object ...> something"
```